examples/misc/lib/language_tour/classes/orchestra.dart

```diff
@@ -51,11 +52,33 @@ class Maestro extends Person
 }
 // #enddocregion Musician-and-Maestro
 
+// #docregion mixin-on
+mixin MusicalPerformer on Musician {
+  // #enddocregion mixin-on
+  bool canDance = true;
+
+  @override
+  void entertainMe() {
+    canDance ? dance() : super.entertainMe();
+  }
+
+  void dance() => print('Dancing');
+  // #docregion mixin-on
+}
+// #enddocregion mixin-on
+
+class SingerDancer extends Musician with MusicalPerformer {
+  SingerDancer(String name) : super(name);
+}
+
 void main() {
   var director = Maestro('Allen');
   director.entertainMe(); // Waving hands
 
   var musician = Musician('Kathy');
   musician.canPlayPiano = true;
   musician.entertainMe(); // Playing piano
+
+  var singerDancer = SingerDancer('Todd');
+  singerDancer.entertainMe(); // Dancing
```

Comment:
You'll probably need to modify `examples/misc/test/language_tour/classes_test.dart`.

Reply:
(And you'll need to `allow_failures` for `dev`)

Reply:
It's failing for stable/raw too, and I can't figure out why.